Makefile

```diff
@@ -175,14 +180,14 @@ tools/firecracker-builder-stamp: tools/docker/Dockerfile.firecracker-builder
 		tools/docker
 	touch $@
 
-$(FIRECRACKER_BIN) $(JAILER_BIN): $(FIRECRACKER_DIR)/Cargo.toml tools/firecracker-builder-stamp
+$(FIRECRACKER_BIN) $(JAILER_BIN): $(FIRECRACKER_DIR)/Cargo.toml $(FIRECRACKER_DIR) tools/firecracker-builder-stamp
```

Comment:
Do we need both `$(FIRECRACKER_DIR)/Cargo.toml` and `$(FIRECRACKER_DIR)` here ? As we pull submodule, `Cargo.toml` will be already there, so we can have just one dependency.

Reply:
No that has no effect right now, this is leftover from something else I was trying but didn't include here, will remove.